### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vpicone**
This pr is to trigger a rebuild to test vercel incremental buiilds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ src
 
 - `yarn install` – install dependencies
 - `yarn dev` – start the development server
-- `yarn dev:clean` – use this if you have cache issues
+- `yarn dev:clean` – use this if you get weird errors that metion `.cache`
```
Comment (joshblack) [inline README.md:26]:
```suggestion
- `yarn dev:clean` – use this if you get weird errors that mention `.cache`
```

**Comment by joshblack** [comment]
Lol ignore that review 😂

**Comment by dakahn** [comment]
i approve this rebuild

**Comment by joshblack** [comment]
Gonna close this out, feel free to re-open if you're still testing!

**Pull request closed without merging**